composer.json

```diff
@@ -21,18 +21,8 @@
         "react/stream": "^0.7.7"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.35",
+        "phpunit/phpunit": "^5.7 || ^4.8.35",
         "clue/block-react": "^1.1",
         "clue/stream-filter": "^1.3"
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "0.5-dev"
-        }
-    },
-    "config": {
-        "platform": {
-            "php": "5.4"
-        }
```

Comment:
Is the removal of this block intendend here? For example, the config.platform setting was introduced in #364.